components/stacks-network/src/orchestrator.rs

```diff
@@ -2335,6 +2430,8 @@ events_keys = ["*"]
                 Err(_e) => {}
             }
             std::thread::sleep(std::time::Duration::from_secs(1));
+            // // let res = self.docker_client.as_ref().unwrap().inspect_container(self.bitcoin_node_container_id.as_ref().unwrap(), None).await;
+            // println!("{:?}", res);
```

Comment:
Is this comment left there on purpose?

Reply:
I'll remove 👍

Reply:
Addressed in https://github.com/hirosystems/clarinet/pull/704/commits/a16a7b783091d9043be30dd9a0c55ad9862cd32a